angular.json

```diff
@@ -39,7 +39,8 @@
             ],
             "scripts": [
               "./node_modules/@popperjs/core/dist/umd/popper.min.js",
-              "./node_modules/bootstrap/dist/js/bootstrap.min.js"
+              "./node_modules/bootstrap/dist/js/bootstrap.min.js",
+              "./node_modules/jquery/dist/jquery.min.js"
```

Comment:
This will cause jQuery to be loaded in all deployments and that's exactly what we want to avoid.

Reply:
Also, why is this needed at all if you're importing from `jquery` in the code directly? Won't it make jQuery loaded twice?